cmd/rhelemeter-server/main.go

```diff
@@ -170,6 +172,7 @@ type Options struct {
 	TracingEndpoint         string
 	TracingEndpointType     string
 	TracingSamplingFraction float64
+	TracingInsecure         bool
```

Comment:
Whats the purpose of this? I would expect the server to push traces into some collector and/or sidecar but perhaps there is something we need to expose?